packages/sdk/test/unit/document/document_size_test.ts

```
@@ -0,0 +1,105 @@
+        str: string;
+      }>
+    >('test-doc');
+
+    doc.update((root) => {
+      root['num'] = 1;
+      root['str'] = 'hello';
+    });
+    assert.deepEqual(doc.getDocSize().live, { data: 14, meta: 120 });
+
+    doc.update((root) => {
+      delete root['num'];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)